### PR TITLE
Speed up job template test by only waiting for project sync after all resources are created

### PR DESCRIPTION
**Pull request by jamestalton**

### CODE REVIEW

File: cypress/e2e/awx/resources/jobTemplates.cy.ts [hunk 1]
```diff
@@ -23,40 +23,42 @@ describe('Job templates form', () => {
...
-      // });
-
-      cy.createAWXCredential({
-        kind: 'machine',
-        organization: organization.id,
-        credential_type: 1,
-      }).then((cred) => {
-        machineCredential = cred;
-      });
+        // cy.createAWXCredential({
```
Comment (AlexSCorey) [inline cypress/e2e/awx/resources/jobTemplates.cy.ts:37]: Please remove the commented code

File: cypress/support/awx-commands.ts [hunk 2]
```diff
@@ -417,15 +417,18 @@ Cypress.Commands.add('deleteAwxUser', (user: User) => {
...
+  ) => {
     cy.awxRequestPost<Partial<Project>, Project>('/api/v2/projects/', {
       name: 'E2E Project ' + randomString(4),
       scm_type: 'git',
       scm_url: 'https://github.com/ansible/ansible-ui',
       ...project,
     }).then((project) => {
-      if (!skipSync) {
-        waitForProjectToFinishSyncing(project.id);
+      if (!options?.skipSync) {
```
Comment (AlexSCorey) [inline cypress/support/awx-commands.ts:430]: I thing this set up is going to bite us.  This is an anti-pattern in controller as a project cannot be used anywhere until it is synced.  At a minimum we should have skipSync default to false.

Comment (jamestalton) [reply cypress/support/awx-commands.ts:430]: it does default to false

File: cypress/e2e/awx/resources/jobTemplates.cy.ts (hunk 1 above)
Comment (jamestalton) [reply cypress/e2e/awx/resources/jobTemplates.cy.ts:37]: the commented code needs to be reenabled after it is made to work

**Pull request closed without merging**